apps/web/tailwind.config.js

```diff
@@ -9,8 +9,11 @@ const {VeniceTheme} = require('./styles/themes')
  */
 module.exports = {
   content: [
-    './{components,pages,screens}/**/*.{ts,tsx}',
-    '../../integrations/*/**/*.tsx',
+    './components/**/*.tsx',
+    './layouts/**/*.tsx',
+    './pages/**/*.tsx',
+    './screens/**/*.tsx',
+    '../../integrations/**/*.tsx',
```

Comment:
Spread them out so adding/removing is easy to see in GH diff.

Reply:
What do you mean by "spreading out"?

Reply:
Instead of having them on one line like `{components,pages,screens}`